storage/scylladb_identity.h

```diff
@@ -197,19 +179,41 @@ status_t db_init_identity_keyspace(db_client_service_t* service, bool need_drop,
  */
 status_t db_get_identity_objs_by_status(db_client_service_t* service, cass_int8_t status,
                                         db_identity_array_t* db_identity_array);
+/**
+ * @brief set time(in seconds) in db_identity_t
+ *
+ * @param[in] obj pointer to db_identity_t
+ * @param[in] ts timestamp to be set into db_identity_t
+ * @return
+ * - SC_OK on success
+ * - SC_TA_NULL/SC_STORAGE_INVAILD_INPUT on error
+ */
+status_t db_set_identity_timestamp(db_identity_t* obj, cass_int64_t ts);
+
+/**
+ * @brief return time(in seconds) in db_identity_t
+ *
+ * @param[in] obj pointer to db_identity_t
+ *
+ * @return
+ * - time(in seconds) on success
+ * - NULL on error
+ */
+cass_int64_t db_ret_identity_timestamp(const db_identity_t* obj);
+
 /**
  * @brief get identity objs with selected id from identity table
  *
  * @param[in] service ScyllaDB client service for connection
- * @param[in] id selected TXN id
+ * @param[in] uuid_string selected uuid string
```

Comment:
The above mentioned "selected id from identity table", but I was not aware of ID.

Reply:
I forget to modify the corresponding comment.